### PR TITLE
feat(v2): implement dangerouslySetInnerHTML

### PR DESCRIPTION
**Pull request by gioboa**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/qwik/src/core/v2/client/vnode-diff.ts [hunk 1]
```diff
@@ -563,6 +568,8 @@ export const vnode_diff = (container: ClientContainer, jsxNode: JSXOutput, vStar
       needsQDispatchEventPatch = createNewElement(jsx, tag);
     }
     // reconcile attributes
+
+    // TODO: attr order
```
Comment (mhevery) [inline packages/qwik/src/core/v2/client/vnode-diff.ts:572]: What is this todo for?

**Review by mhevery — APPROVED** [review]

File: packages/qwik/src/core/render/execute-component.ts [hunk 2]
```diff
@@ -239,3 +239,9 @@ export const shouldWrapFunctional = (res: unknown, node: JSXNode) => {
 export const static_listeners = 1 << 0;
 export const static_subtree = 1 << 1;
 export const dangerouslySetInnerHTML = 'dangerouslySetInnerHTML';
+export const keysToSkip = [dangerouslySetInnerHTML];
+
+export function getAttributeValueByKey(attrs: (string | null)[], key: string) {
```
[the remark this answers is not in the document]
Comment (Varixo) [reply packages/qwik/src/core/render/execute-component.ts]: hmm, I think this function has been removed before

File: packages/qwik/src/core/v2/client/vnode-diff.ts (hunk 1 above)
Comment (gioboa) [reply packages/qwik/src/core/v2/client/vnode-diff.ts:573]: Removed

File: packages/qwik/src/core/render/execute-component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (gioboa) [reply packages/qwik/src/core/render/execute-component.ts]: it's resolved

**Comment by gioboa** [comment]
image

**Pull request merged**